plugins/statistics.koplugin/main.lua

```diff
@@ -238,7 +241,13 @@ function ReaderStatistics:onDocumentRerendered()
     --   - we only then update self.data.pages=254 as the new page count
     -- - 5 minutes later, on the next insertDB(), (153, now-5mn, 42, 254) will be inserted in DB
 
-    local new_pagecount = self.document:getPageCount()
+    local new_pagecount
+    if ReaderStatistics:usePageMapForPageNumbers() then
+        local __, new_pagecount_t = self.document:getPageMapCurrentPageLabel()
+        new_pagecount = new_pagecount_t
```

Comment:
I think `select(2, ...` is good here.

Reply:
and I think the reference pages count is the 3rd item - the 2nd item is the sequential index of the current page.
This is what can and should be saved in the db.